### PR TITLE
Changed selected radio to blue

### LINKED ISSUE #859: Selected radio button should be blue instead of black
The selected radio button in the latest version is black instead of blue.

image

### PR DESCRIPTION
**Pull request by agliga**
## Description
In the `scripts/image-config.json` added `"icon-radio-checked": "#3665F2"` and reran the generate scripts build.

## Context
As noticed, there are some new DS4 icons in the `base64-variables.less`. However, notice no build files are changed since it's not being used (I can remove if you think it's polluting the repo). In the future we should use it.

## References
Fixes #859

## Screenshots
image

### CODE REVIEW

**Comment by agliga** [comment]
> Actually, hold on. Should we also keep a black version and have a filled version? Like we do with confirmation, attention, etc?

This would be a breaking change then right? Because the default is colored. 
We could add a new icon type (and easy to do), which might be better I would think. But maybe we can have a new icon type once we come up with a solution for the svg fill colors.

**Comment by ianmcburnie** [comment]
It doesn't necessarily have to be a breaking change. Pages with the current icon name (icon-radio-unchecked) would stay with a black radio button (not the end of the world). Pages that opt into the new name (icon-radio-filled-unchecked) will get the blue radio button.

I'm trying to stay consistent with how the attention, confirmation, information have been done. Whereby "black" is the default, and `-filled` gets the coloured version. I think it may have been better to flip this around and have the coloured version as the default, and the black version to require a different name. Do the black versions of those icons ever get used?

**Comment by agliga** [comment]
> It doesn't necessarily have to be a breaking change. Pages with the current icon name (icon-radio-unchecked) would stay with a black radio button (not the end of the world). Pages that opt into the new name (icon-radio-filled-unchecked) will get the blue radio button.
> 
> I'm trying to stay consistent with how the attention, confirmation, information have been done. Whereby "black" is the default, and `-filled` gets the coloured version. I think it may have been better to flip this around and have the coloured version as the default, and the black version to require a different name. Do the black versions of those icons ever get used?

I will do the same for checkbox then since it will be more constant that way

**Comment by ianmcburnie** [comment]
Actually, instead of creating a new symbol with fill, could we just do something like this instead: 

```
.checkbox__icon svg {
  fill: blue
}
```

The background icon version will of course need a hardcoded base64 version with blue.
```

**Comment by ianmcburnie** [comment]
@agliga Ok, having now just realised this ONLY refers to the background version of the icon, NOT the foreground icon too, let's just quickly make the background radio icon blue and not worry about anything else I mentioned :-) I may come back to that in v10 though.

File: src/less/variables/ds6/base64-variables.less [hunk 1]
```diff
@@ -93,7 +93,7 @@
...
-@icon-radio-checked-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLXJhZGlvLWNoZWNrZWQiIHZpZXdCb3g9IjAgMCAyNCAyNCI+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTEyIDBDNS4zNzMgMCAwIDUuMzczIDAgMTJzNS4zNzMgMTIgMTIgMTIgMTItNS4zNzMgMTItMTJBMTEuOTk4IDExLjk5OCAwIDAwMTIgMHptMCAyMkM2LjQ3NyAyMiAyIDE3LjUyMyAyIDEyUzYuNDc3IDIgMTIgMnMxMCA0LjQ3NyAxMCAxMC00LjQ3NyAxMC0xMCAxMHptNy0xMGE3IDcgMCAxMS0xNCAwIDcgNyAwIDAxMTQgMHoiIGZpbGw9IiMxMTE4MjAiLz4KPC9zdmc+";
+@icon-radio-checked-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLXJhZGlvLWNoZWNrZWQiIHZpZXdCb3g9IjAgMCAyNCAyNCI+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgZD0iTTEyIDBDNS4zNzMgMCAwIDUuMzczIDAgMTJzNS4zNzMgMTIgMTIgMTIgMTItNS4zNzMgMTItMTJBMTEuOTk4IDExLjk5OCAwIDAwMTIgMHptMCAyMkM2LjQ3NyAyMiAyIDE3LjUyMyAyIDEyUzYuNDc3IDIgMTIgMnMxMCA0LjQ3NyAxMCAxMC00LjQ3NyAxMC0xMCAxMHptNy0xMGE3IDcgMCAxMS0xNCAwIDcgNyAwIDAxMTQgMHoiIGZpbGw9IiMzNjY1RjIiLz4KPC9zdmc+";
```
Comment (seangates) [inline src/less/variables/ds6/base64-variables.less]: The blue color is one digit off. You have `#3665F2`, but it's `#3665F3` in our [color variable](https://github.com/eBay/skin/blob/9.2.0/dist/variables/ds6/color-variables.less#L42).

File: scripts/image-config.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
     "base64Config": {
         "ds6Color": "#111820",
         "overrides": {
-            "icon-checkbox-checked": "#3665F2"
+            "icon-checkbox-checked": "#3665F2",
+            "icon-radio-checked": "#3665F2"
```
Comment (seangates) [inline scripts/image-config.json]: Same as the other one.

**Pull request merged**